modules/okenetwork/outputs.tf

```
@@ -3,6 +3,7 @@
 
 output "subnet_ids" {
   value = map(
+    "cp", join(",", oci_core_subnet.cp.*.id),
```

This splat expression syntax is legacy (attribute-only). You should switch to the current splat expression syntax.

```suggestion
    "cp", join(",", oci_core_subnet.cp[*].id),
```

Same suggestion from line 6 to 9.